code/modules/mob/emote.dm

```diff
@@ -1,3 +1,18 @@
+#define EMOTE_COOLDOWN 50

```

Comment:
being nitpicky here, but you might want to add in a comment that this is in deciseconds.